### PR TITLE
Feature/random cleanup

### PR DESCRIPTION
**Pull request by neophob**
- rename main.js to index.js
- cleanup readme
- bump used sdl lib and go version
- remove unused const
- try to add lint step... which does not work

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,37 +50,40 @@ build-wasm: wasm-common
 	@env GOARCH=wasm GOOS=js go build -o "$(DISTDIR)/lib.wasm" $(WASMDIR)
 	@echo "  DONE! run devserver in the dist directory"
 
-## build-wasm: builds the wasm app
-build-wasm-release: wasm-common
-	@echo "  >  BUILD WASM release version"
-	@env GOARCH=wasm GOOS=js go build -o "$(DISTDIR)/lib.wasm" $(RELEASE) $(WASMDIR)
```
Comment (neophob) [inline Makefile:56]: there was no difference between release and debug build!

File: go.mod [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module github.com/neophob/ganother-world
 
-go 1.13
+go 1.14
```
Comment (pfiaux) [inline go.mod:3]: Guess I have to update :)

File: Makefile [hunk 3]
```diff
@@ -50,37 +50,40 @@ build-wasm: wasm-common
...
-	@go fmt $(PACKAGES)
+	@gofmt -w .
 
 ## test: run unit tests
 test:
 	@go test -cover -v $(PACKAGES_TO_TEST)
 
+## lint: static analyze source
+lint:
+	@env GOARCH=wasm GOOS=js go vet ./...
```
[the remark this answers is not in the document]
Comment (pfiaux) [reply Makefile:63]: Minor: add lint to the github actions/workflow

File: go.mod (hunk 2 above)
Comment (neophob) [reply go.mod:3]: bleeding edge!

File: Makefile (hunk 3 above)
[the remark this answers is not in the document]
Comment (neophob) [reply Makefile:63]: no, I have the same issue, see https://github.com/veandco/go-sdl2/issues/461

if this is resolved, i'll add the github action

[the remark this answers is not in the document]
Comment (neophob) [reply Makefile:63]: AH! `GOARCH=wasm GOOS=js` guess i need ti differenciate wasm and sdl build

**Pull request merged**